sematic/ee/plugins/publishing/github/publisher.py

```diff
@@ -0,0 +1,67 @@
+"""
+The GitHub Publisher plugin implementation.
+"""
+# Standard Library
+import logging
+from typing import Any, Type, cast
+
+# Sematic
+# Required so that the endpoint will get registered when this plugin is used & imported.
+import sematic.ee.plugins.publishing.github.endpoints  # noqa: F401
+from sematic.abstract_plugin import (
+    SEMATIC_PLUGIN_AUTHOR,
+    AbstractPlugin,
+    AbstractPluginSettingsVar,
+    PluginVersion,
+)
+from sematic.db.models.resolution import Resolution, ResolutionStatus
+from sematic.db.queries import get_run
+from sematic.ee.plugins.publishing.github.check import (
+    COMMIT_CHECK_PREFIX,
+    GitHubPublisherSettingsVar,
+    check_commit,
+)
+from sematic.plugins.abstract_publisher import AbstractPublisher
+
+logger = logging.getLogger(__name__)
+
+
+_PLUGIN_VERSION = (0, 1, 0)
+
+
+class GitHubPublisher(AbstractPublisher, AbstractPlugin):
```

Comment:
How about we expose this in `plugins/publishing/github/__init__.py` so that users don't have to add the `publisher` bit (which is somewhat private implementation) in the Helm chart and settings.

Reply:
Users will never be interacting with this path directly. Since it's a server plugin, they will be turning it on/off via the helm value, which is just `github.enabled`. So this is probably not worth it.